storage/badger/guarantees_test.go

```diff
@@ -16,21 +16,21 @@ import (
 func TestGuaranteeStoreRetrieve(t *testing.T) {
 	unittest.RunWithBadgerDB(t, func(db *badger.DB) {
 		metrics := metrics.NewNoopCollector()
-		store := badgerstorage.NewGuarantees(metrics, db, 1000)
+		store := badgerstorage.NewGuarantees(metrics, db, 1000, 1000)
 
 		// abiturary guarantees
 		expected := unittest.CollectionGuaranteeFixture()
 
 		// retrieve guarantee without stored
-		_, err := store.ByCollectionID(expected.ID())
+		_, err := store.ByCollectionID(expected.CollectionID)
 		require.ErrorIs(t, err, storage.ErrNotFound)
 
 		// store guarantee
 		err = store.Store(expected)
 		require.NoError(t, err)
 
 		// retreive by coll idx
-		actual, err := store.ByCollectionID(expected.ID())
+		actual, err := store.ByID(expected.ID())
 		require.NoError(t, err)
 		require.Equal(t, expected, actual)
```

Comment:
```suggestion
		require.Equal(t, expected, actual)
		
		actual, err = store.ByCollectionID(expected.CollectionID)
		require.NoError(t, err)
		require.Equal(t, expected, actual)
```

Testing both methods in each case (exists / not exists)